openapi/ga4gh-tool-discovery.yaml

```diff
@@ -763,8 +767,8 @@ definitions:
         description: >- 
           A production (immutable) tool version is required to have a hashcode. Not required otherwise, but might be useful to detect changes. 
         example:
-          - checksum: ea2a5db69bd20a42976838790bc29294df3af02b
-            type: sha1
+          - checksum: 77af4d6b9913e693e8d0b4b294fa62ade6054e6b2f1ffb617ac955dd63fb0182
```

Comment:
The example on line 495 has quotes; this one doesn't. But they're optional in YAML, unless they're required.